docker/thirdparties/docker-compose/hive/gen_env.sh.tpl

```diff
@@ -23,9 +23,11 @@
 set -eo pipefail
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
-
-FS_PORT=8120
-HMS_PORT=9183
+eth0_num=$(ifconfig -a|grep flags=|grep -n ^eth0|awk -F ':' '{print $1}')
```

Comment:
Not all hosts have `eth0`

Reply:
1. I give a defult 127.0.0.1 value to IP_HOST，this can satisfy the needs of most people
2.If you cannot obtain the corresponding IP address through eth0, the system prompts you to set the IP address as required